media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format. flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="https://flac-tools.sourceforge.net/"
```

Comment:
Oops, the https is for SOURCE_URI, not for HOMEPAGE.

BTW, don'y you want to drop the autoreconf in favor of the aclocal + automake --missing ?
autoreconf calls other tools which are not needed here.